src/backend/base/langflow/template/template/base.py

```diff
@@ -56,6 +56,11 @@ def from_dict(cls, data: dict) -> "Template":
                     _input = Input(**value)
 
                 data["fields"].append(_input)
+
+        # Necessary for components with no inputs(?)
+        if "fields" not in data:
+            data["fields"] = []
```

Comment:
This is my only follow up - still not entirely sure if this breaks anything else.

Reply:
Yes. fields is required by the Template

Reply:
Okay - and I am assuming that `fields` is not populated in `data` for this new component because there are no `inputs`. In that case, this is good to merge?